holoviews/core/operation.py

```diff
@@ -103,6 +103,10 @@ class ElementOperation(Operation):
        first component is a Normalization.ranges list and the second
        component is Normalization.keys. """)
 
+    link_inputs = param.Boolean(default=False, doc="""
+         Whether a plot created from the output of this operation
+         should be linked to the inputs of this operation.""")
+
```

Comment:
Needs a *lot* more explanation as almost none of the required concepts exist at this level:

1. An operation doesn't create plots, it generates a data-structure.
2. Specifically, linking is about streams which relates to DynamicMap so this should say something about when the operation is dynamic.
3. Linked streams only exist for specific backends.

As a very rough suggestion, it should be something along the lines of:

'If the operation is dynamic, whether or not linked streams should be transferred from the operation inputs for backends that support linked streams. For example ... '